Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -429,6 +429,14 @@ open class KMConversationViewController: ALKConversationViewController {
         viewModel.prepareController()
         ALMessageService.syncMessages()
     }
+    
+    open override func addLanguageToMetadata(language: String) {
```

Comment:
what's the use of this?

Reply:
it will add selected language code to the message metadata. its requested by the mindgraph

Reply:
Can they not do it from outside by observing the language selection event and then passing it in metadata through config? Unless it's required for all.

Reply:
they wanted it by default from sdk itself. Shall i add customisation flag to handle this only for them ? @mukesh-komm

Reply:
We can leave it if we have already committed, otherwise observing the event would have been better.